packages/controllers/file_upload.coffee

```diff
@@ -0,0 +1,19 @@
+Template.fileUpload.onCreated ->
+  @fileName = new ReactiveVar('No file selected')
+  @hasFile = new ReactiveVar(false)
+
+Template.fileUpload.helpers
+  fileName: ->
+    Template.instance().fileName.get()
+  hasFile: ->
+    if Template.instance().hasFile.get()
+      'has-file'
+
+Template.fileUpload.events
+  'change .btn-file': (event, template) ->
+    template.fileName.set(event.target.value.replace('C:\\fakepath\\',''))
```

Comment:
Oh that's interesting. Does HTML5 actually specify what the fake path should be? And can you put in a comment about this?